packages/gui/src/data/properties.ts

```diff
@@ -1095,6 +1095,9 @@ export const properties: Record<string, PropertyData> = {
     type: 'keyword',
     keywords: ['normal', 'sub', 'super'],
   },
+  fontVariationSettings: {
+    type: 'none'
+  },
```

Comment:
This shouldn't be part of the list of properties which a user can select, but because variable fonts add this as a key the Control component attempts to render it, but `getPrimitiveInput` breaks. This patches that break.